features/step_definitions/help_relinking.rb

```diff
@@ -17,15 +17,21 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 #++
 
-Given /^the help link setting points to the project (.+)$/ do |project_name|
-  project_name.gsub!("\"", '')
-  p = Project.find_by_name(project_name)
+Given(/^the help link setting points to help page$/) do
   new_values = Setting.plugin_openproject_help_link
-  new_values['help_link_target'] = "/projects/#{p.identifier}"
+  new_values["help_link_target"] = OpenProject::Info.help_url
   Setting.plugin_openproject_help_link = new_values
```

Comment:
another good way is to make your test fail before you make it pass. in this case: configure a different url and see that it doesn't go to the help page (as the scenario title suggests)